crates/forge/bin/cmd/test/mod.rs

```diff
@@ -634,6 +639,14 @@ impl TestArgs {
         if self.rerun {
             filter.test_pattern = last_run_failures(config);
         }
+        if filter.path_pattern.is_some() {
+            if self.path.is_some() {
+                println!("Can not supply both --match-path and |path|");
+                std::process::exit(0);
```

Comment:
It is better to use `panic!` here, exit code `0` indicates success

Reply:
Done.